api/api.go

```diff
@@ -418,7 +418,7 @@ func LogCertificate(w http.ResponseWriter, cert *x509.Certificate) {
 				if len(val.CredentialID) > 0 {
 					m["provisioner"] = fmt.Sprintf("%s (%s)", val.Name, val.CredentialID)
 				} else {
-					m["provisioner"] = val.Name
+					m["provisioner"] = string(val.Name)
```

Comment:
@maraino is this a result of a recent change to how we store the oid extension?

Reply:
No, it looks like it has been like this forever. 

For most of the provisioners is not a problem, but provisioners like X5C that do not have a CredentialID shows `provisioner="[88 53 67]` instead of `provisioner="X5C"`